packages/faency/src/components/Avatar.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useContext, forwardRef } from 'react'
+import { Avatar as AvatarPrimitive, AvatarProps as AvatarPrimitiveProps } from '@modulz/primitives'
+import { ThemeContext } from 'styled-components'
+import useDarkMode from 'use-dark-mode'
```

Comment:
`use-dark-mode` shouldn't be used inside faency. We are trying to keep components stateless. As a workaround if you want always white or always black why not using hexa codes?

Reply:
sure, makes sense!